src/main/java/spoon/reflect/path/CtPathStringBuilder.java

```diff
@@ -145,7 +145,8 @@ private String parseArgumentValue(Tokenizer tokenizer, String argName, CtPathEle
 				}
 			} else if ("]".equals(token) || ";".equals(token)) {
 				//finished reading of argument value
-				pathElement.addArgument(argName, argValue.toString());
+				//[fix bug]:AbstractPathElement.getArguments([constructor with no parameters])
```

Comment:
I am not sure what this is doing here, I will need to have a look at it later. This also feels like it should be a different PR and have a test case :)

Reply:
Yes it's a small bug in method `AbstractPathElement.getArguments()`, when a default constructor in, it produces redundant `)`.